README.rst

```diff
@@ -2,9 +2,8 @@
 ``pipreqs`` - Generate requirements.txt file for any project based on imports
 =============================================================================
 
-.. image:: https://img.shields.io/travis/bndr/pipreqs.svg
-        :target: https://travis-ci.org/bndr/pipreqs
-
+.. image:: https://github.com/bndr/pipreqs/actions/workflows/tests.yml/badge.svg?branch=master
```

Comment:
I set the tracking to tests performed on the master branch, a broken test in the next branch or in a random PR doesnt seem to be very informative or relevant to users since unless they manually instal pipreqs they will get the master branch through pypi